internal/pkg/cli/dockerfile.go

```diff
@@ -51,7 +53,9 @@ func listDockerfiles(fs afero.Fs, dir string) ([]string, error) {
 		}
 	}
 	if len(directories) == 0 {
-		return nil, fmt.Errorf("no Dockerfiles found within %s or a sub-directory level below", dir)
+		return nil, &dockerfile.ErrDockerfileNotFound{
+			Dir: dir,
+		}
```

Comment:
nit: Since this error is not returned from the `dockerfile` pkg itself, I think it should be moved to the `cli` package and be made private

Reply:
Okay! Moved it to `cli/dockerfile.go`.